### DISCUSSION TITLE
希望有錯題本

### DISCUSSION THREAD

**Discussion by mowmitw** [SiongSng/cap-countdown#118] [提點子]
有錯題本就會要訂正，希望可以把筆記功能改成可以放圖片的一整個方塊在題目的最下面，希望可以把在模擬歷屆會考裡面的筆記連同題目一起到錯題本，方便訂正時回想解題思路。
(希望每日一題也可以有收藏的選項)

**Comment by SiongSng** [comment]
感謝你的回饋，我覺得這個點子不錯 ✨，我會儘快找時間實施。
